attributes/dotnet_agent.rb

```diff
@@ -3,6 +3,6 @@
 default['appdynamics']['dotnet_agent']['checksum'] = '7acb3756147a1d5a13c49b107a890ea56a8eb4099fd793e498e34b6f0b5962dc' #nil
 default['appdynamics']['dotnet_agent']['install_dir'] = 'C:\Program Files\Appdynamics'
 default['appdynamics']['dotnet_agent']['logfiles_dir'] = 'C:\DotNetAgent\Logs'
-
+default['appdynamics']['dotnet_agent']['standalone_apps'] = nil #'a.exe,b.exe'
```

Comment:
if there are multiple exe's like a.exe & b.exe both will end up with the same app_name/tier_name in the controller even though they are different exe's.  can we change the standalone_apps to an array that includes the 3 possible properties? standalone-application executable, command-line, and tier name

example config from the docs
    <standalone-applications>
        <standalone-application executable="MyWindowsService.exe" command-line="-x">
           <tier name="Windows Service Tier" />
        </standalone-application>
        <standalone-application executable="MyStandaloneApp.exe">
           <tier name="Standalone Tier" />
        </standalone-application>
    </standalone-applications>
https://docs.appdynamics.com/display/PRO41/Instrument+Windows+Services+and+Standalone+Applications